File: 1-Iniciante/ex1035.js
// entrada
let input = require('fs').readFileSync('dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd
let [a, b, c, d] = input.split(' ').map(n => parseInt(n));

// processamento
let aceito = false;
if(b > c){
    if(d > a){
        if(c + d > a + b){
            if(c > -1 && d > -1){
                if(a % 2 === 0){
                    aceito = true;
                }
            }
        }
    }
}

// saida
console.log(aceito ? "Valores aceitos" : "Valores nao aceitos");

File: 1-Iniciante/ex1035.py
#entrada
a, b, c, d = str(input()).split()
a = int(a)
b = int(b)
c = int(c)
d = int(d)

#processamento/saída
if b > c and d > a and c + d > a + b and c >= 0 and d >= 0 and a % 2 == 0:
    print("Valores aceitos")
else:
    print("Valores nao aceitos")
